.github/workflows/pr-check.yaml

```diff
@@ -17,10 +17,14 @@
 
 name: pr-check
 
-on: 
+on:
   pull_request:
     types: [labeled, synchronize, opened, reopened]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
looks like sometimes the pattern is 
```
${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Reply:
https://github.com/pytorch/pytorch/blob/main/.github/workflows/pull.yml#L17
https://github.com/jaegertracing/jaeger/blob/main/.github/workflows/fossa.yml#L11

Reply:
What would be the difference compare to the value given in the github documentation ?

Reply:
you can change the branch of the PR I guess ?